errors.go

```diff
@@ -9,6 +9,12 @@ import (
 )
 
 var (
+	// ErrComputeSystemDoesNotExist is an error returned when the compute system being operated on no longer exists
+	ErrComputeSystemDoesNotExist = syscall.Errno(0xc037010e)
+
+	// ErrElementNotFound is an error returned when the object being referenced does not exist
+	ErrElementNotFound = syscall.Errno(0x490)
```

Comment:
I've only seen it being returned when a process either has already exited, or does not exit, and a call is made to modify the process, or get properties, but it looks like it could also be thrown if a sandbox is not found, and I may be potentially missing more.